master/buildbot/worker/openstack.py

```diff
@@ -231,39 +234,58 @@ def _getImage(self, build):
         # function should return the image's UUID to use.
         image = self.image
         if callable(image):
-            image_uuid = image(self.novaclient.images.list())
+            # novaclient has renamed images to glance
+            if hasattr(self.novaclient, "glance"):
+                image_uuid = image(self.novaclient.glance.list())
+            else:
+                image_uuid = image(self.novaclient.images.list())
         else:
             image_uuid = yield build.render(image)
         return image_uuid
 
+    @defer.inlineCallbacks
+    def _getFlavor(self, build):
```

Comment:
Since this is private, let's name it as `_get_flavor`.

Reply:
This is not consistent with the rest of the class, I can rename it but it will be different from others.

Reply:
My recollection is Buildbot follows Twisted's style and that would be `_getFlavor`.

Reply:
We've changed this recently, see http://docs.buildbot.net/latest/developer/style.html

Reply:
@p12tic so what should be done? Fix the entire file, just this instance or?

Reply:
I think we can use the new approach on the newly added code only. This does introduce inconsistency, but as the time goes, more and more of Buildbot will migrate and this will become less of an issue.